core/src/main/java/org/sonarsource/sonarlint/core/ConnectedSonarLintEngineImpl.java

```diff
@@ -140,7 +141,8 @@ private PluginsLoadResult loadPlugins() {
     pluginsToLoadByKey.putAll(globalConfig.getEmbeddedPluginPathsByKey());
     Set<Path> plugins = new HashSet<>(pluginsToLoadByKey.values());
 
-    var config = new Configuration(plugins, globalConfig.getEnabledLanguages(), Optional.ofNullable(globalConfig.getNodeJsVersion()), globalConfig.getAllowedPluginIds());
+    var config = new Configuration(plugins, globalConfig.getEnabledLanguages(), Optional.ofNullable(globalConfig.getNodeJsVersion()),
+      DataflowBugDetection.getPluginAllowList(globalConfig.isDataflowBugDetectionEnabled()));
```

Comment:
Is this parameter still relevant with the change you made in PluginServiceImpl? Looks duplicated to me 🤔

Reply:
I tried to push the feature flag even further down; we still need it both on the engine side _and_ backend side AFAICT.